src/legend.js

```diff
@@ -160,12 +160,16 @@ export const Legend = Class.create( {
      * @param {String} name The human-readable object name or description
      * @return {HTMLLIElement} List element to be insert in the legend
      */
-  _generateElement: function(id, name) {
+  _generateElement: function(id, name, isObsolete) {
     var color = this.getObjectColor(id);
-    var item = new Element('li', {'class' : 'disorder '+'drop-'+this._getPrefix(), 'id' : this._getPrefix() + '-' + id}).update(new Element('span', {'class' : 'disorder-name'}).update(name));
+    var obsoleteClass = isObsolete ? 'obsolete ' : '';
+    var item = new Element('li', {'class' : 'disorder ' + obsoleteClass + 'drop-'+this._getPrefix(), 'id' : this._getPrefix() + '-' + id}).update(new Element('span', {'class' : 'disorder-name'}).update(name));
     var bubble = new Element('span', {'class' : 'disorder-color'});
+    var tooltiptext = new Element('span', { class: 'tooltiptext' }).insert(
+      'This HPO term is obsolete. See the HPO terms tab for details.'
+    );
     bubble.style.backgroundColor = color;
-    item.insert({'top' : bubble});
+    item.insert({'top' : bubble}).insert(tooltiptext);
```

Comment:
Just so I understand, are we always adding the tooltip to say it's obsolete, and the CSS is handling whether it's shown on hover or not? We could conditionally add the tooltip text dependent on if it's obsolete instead? Up to you if you want to change it

Reply:
I think it's sensible to only add it when needed, so now conditionally adding it when `isObsolete` is true.